test/wc.js

```diff
@@ -0,0 +1,146 @@
+});
+
+test('multiple files', t => {
+    const result = shell.wc('test/resources/head/file2.txt', 'test/resources/head/file1.txt');
+    t.falsy(shell.error());
+    t.is(result.code, 0);
+    t.is(result.toString(),
+        bottomOfFile2
+        .slice(0, 10)
+        .reverse()
+        .concat(bottomOfFile1.slice(0, 10).reverse())
+        .join('\n') + '\n');
+});
+
+test('multiple files, array syntax', t => {
+    const result = shell.wc(['test/resources/head/file2.txt', 'test/resources/head/file1.txt']);
+    t.falsy(shell.error());
+    t.is(result.code, 0);
+    t.is(result.toString(),
+        bottomOfFile2
+        .slice(0, 10)
+        .reverse()
+        .concat(bottomOfFile1.slice(0, 10).reverse())
+        .join('\n') + '\n');
+});
+
+test('reading more lines than are in the file (no trailing newline)', t => {
+    const result = shell.wc('test/resources/file2', 'test/resources/file1');
+    t.falsy(shell.error());
+    t.is(result.code, 0);
+    t.is(result.toString(), 'test2\ntest1'); // these files only have one line (no \n)
+});
+
+test('reading more lines than are in the file (with trailing newline)', t => {
+    const result = shell.wc('test/resources/head/shortfile2', 'test/resources/head/shortfile1');
+    t.falsy(shell.error());
+    t.is(result.code, 0);
+    t.is(result.toString(), 'short2\nshort1\n'); // these files only have one line (with \n)
+});
+
+test('Globbed file', t => {
+    const result = shell.wc('test/resources/head/file?.txt');
+    t.falsy(shell.error());
+    t.is(result.code, 0);
+    t.is(result.toString(),
+        bottomOfFile1
+        .slice(0, 10)
+        .reverse()
+        .concat(bottomOfFile2.slice(0, 10).reverse())
+        .join('\n') + '\n');
+});
+
+test('With `\'-n\' <num>` option', t => {
+    const result = shell.wc('-n', 4, 'test/resources/head/file2.txt',
+        'test/resources/head/file1.txt');
+    t.falsy(shell.error());
+    t.is(result.code, 0);
+    t.is(result.toString(),
+        bottomOfFile2
+        .slice(0, 4)
+        .reverse()
+        .concat(bottomOfFile1.slice(0, 4).reverse())
+        .join('\n') + '\n');
+});
+
+test('With `{\'-n\': <num>}` option', t => {
+    const result = shell.wc({ '-n': 4 }, 'test/resources/head/file2.txt',
+        'test/resources/head/file1.txt');
+    t.falsy(shell.error());
+    t.is(result.code, 0);
+    t.is(result.toString(),
+        bottomOfFile2
+        .slice(0, 4)
+        .reverse()
+        .concat(bottomOfFile1.slice(0, 4).reverse())
+        .join('\n') + '\n');
+});
+
+test('negative values are the same as positive values', t => {
+    const result = shell.wc('-n', -4, 'test/resources/head/file2.txt',
+        'test/resources/head/file1.txt');
+    t.falsy(shell.error());
+    t.is(result.code, 0);
+    t.is(result.toString(),
+        bottomOfFile2
+        .slice(0, 4)
+        .reverse()
+        .concat(bottomOfFile1.slice(0, 4).reverse())
+        .join('\n') + '\n');
+});
+
```

Comment:
Remove extra newline (just delete line 146)

Reply:
I opened #816 to catch this automatically. Once that PR lands, you can rebase off master and fix this with `npm run lint -- --fix`.